lib/kafka/connection.rb

```diff
@@ -156,8 +227,8 @@ def write_request(request, notification)
     #   a given Decoder.
     #
     # @return [nil]
-    def read_response(response_class, notification)
-      @logger.debug "Waiting for response #{@correlation_id} from #{to_s}"
+    def read_response(expected_correlation_id, notification)
```

Comment:
this correlation id stuff means that we can still send consumer group related requests to the broker over the same channel, right?